app-frontend/src/app/components/tools/diagramContainer/diagramContainer.controller.js

```diff
@@ -2,6 +2,64 @@
 
 const Map = require('es6-map');
 
+const defaultBreakpoints = [{
```

Comment:
This file is starting to get scary huge; it's not critical now, but it would be good to think of ways that we can trim this down.

Reply:
yea, I'll move the default stuff into a json file, we're gonna be removing it later anyways